content/TruenasApps/StableApps/InstallNextCloudMedia.md

```diff
@@ -15,181 +15,301 @@ keywords:
+{{< hint info >}}
+This basic procedure covers the required Nextcloud app settings.
+For optional settings, see [Understanding App Installation Wizard Settings](#understanding-app-installation-wizard-settings).
+{{< /hint >}}
 
-   When set, the **Installed Applications** screen displays **Apps Service Running** on the top screen banner.
+{{< include file="/static/includes/apps/AddMultipleAppInstancesAndNaming.md" >}}
 
-1. Click **Discover Apps** and then locate the **Nextcloud** app.
-   Change the **Sort** to **App Name**, then type **Nextcloud** in the search field to display the app widget.
+{{< include file="/static/includes/apps/LocateAndOpenInstallWizard.md" >}}
 
-   {{< trueimage src="/images/SCALE/Apps/LocateNextcloudAppWidget.png" alt="Locate Nextcloud App Widget" id="Locate Nextcloud App Widget" >}}
+{{< trueimage src="/images/SCALE/Apps/InstallNextcloudScreen.png" alt="Install Nextcloud Screen" id="Install Nextcloud Screen" >}}
 
-   Click on the widget to open the **Nextcloud** details screen, then click **Install**.
-   If this is the first application installed, TrueNAS displays a dialog about configuring apps.
+{{< include file="/static/includes/apps/AppsWizardAppNameAndVersion.md" >}}
 
-   {{< trueimage src="/images/SCALE/Apps/AppsInformationDialog.png" alt="Apps Information Dialog" id="Apps Information Dialog" >}}
+Next, enter the **Nextcloud Configuration** settings.
+{{< expand "Entering Nextcloud Configuration Settings" "v" >}}
+For a basic installation, you can accept default values, but enter the following values:
+Enter the name of the administration user in **Admin User** and **Admin Password**.
+If using an existing Nextcloud account, enter the administration credentials for that account or enter new to create new Nextcloud user account credentials.
 
-   Click **Confirm** then **Agree** to close the dialog and open the **Nextcloud** details screen opens.
+{{< trueimage src="/images/SCALE/Apps/InstallNextcloudConfig1.png" alt="Nextcloud Configuration Username" id="Nextcloud Configuration Username" >}}
 
-   {{< trueimage src="/images/SCALE/Apps/NextcloudAppDetailsScreen.png" alt="Nextcloud Details Screen" id="Nextcloud Details Screen" >}}
+Select the APT packages you want to use. Nextcloud requires ffmpeg and smbclient. Click **Add** to the right of **APT Packages** twice to add two sets of **Package** fields.
+Select **ffmpeg** in one, and **smbclient** in the other.
+If also selecting **ocrmypdf**, also set the **Tesseract Language Codes** option to use.
+Click **Add** to show the **Language** field then enter either **chi-sim** for simplified Chinese or **eng** for English.
+See [Nextcloud Configuration Settings](#nextcloud-configuration-settings) below for more information.
 
-   Click **Install** to open the **Install Nextcloud** wizard.
+Enter either the fully qualified domain name or the IP address and port for your TrueNAS system in **Host** as *##.###.###.##:port#* or *my.domain.com:port#*.
```

Comment:
I think the # has the potential to add confusion rather than help

```suggestion
Enter either the fully qualified domain name or the IP address and port for your TrueNAS system in **Host** as *##.###.###.##:port* or *my.domain.com:port*.
```